staging/src/k8s.io/sample-controller/main.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+
+	"github.com/golang/glog"
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
+
+	"k8s.io/sample-controller/pkg/signals"
+)
+
+func main() {
+	flag.Parse()
```

Comment:
Not sure what is the best practice here but we can we create the client config (from `kubeconfig`) here? Since it's part of the setup and not a part of the "controller" as such.

Reply:
👍  done